### ISSUE TITLE
tpot 0.11.5 not printing result for each generation anymore

### ISSUE THREAD

**Issue by fcoppey** [EpistasisLab/tpot#1082]
I can rapidly see the generation result then it is overwritten by the progress bar...
Am I the only one?

**Comment by weixuanfu** [comment]
I found this issue when using ipython and I did not find this issue when using jupyter notebook. I pushed a path to development branch to fix the issue and we will release a new minor release with the patch.

You may try the dev branch via the command below:

```shell
pip install --upgrade --no-deps --force-reinstall git+https://github.com/EpistasisLab/tpot.git@development
```

**Comment by fcoppey** [comment]
still not in release?

**Comment by weixuanfu** [comment]
@fcoppey not yet, we will release it later this Month.

**Comment by weixuanfu** [comment]
The issue is fixed in v0.11.6.
